__tests__/.eslintrc

```diff
@@ -3,6 +3,10 @@
   "rules": {
     // vitest currently allows for __dirname, require, etc.
     // see https://github.com/vitest-dev/vitest/issues/2841
-    "unicorn/prefer-module": "off"
+    "unicorn/prefer-module": "off",
+
+    "@vitest/require-mock-type-parameters": "off",
+    "@vitest/padding-around-expect-groups": "off",
+    "@vitest/padding-around-all": "off",
```

Comment:
why are these disabled?

Reply:
disabled the first one because typing `vi.fn()` was becoming a huge pain.

disabled the last two because it wants to update a whole bunch of tests to add padding around `expect()` calls. can enable them if you like

Reply:
nah im always pro whitespace. will enable it